images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -257,4 +257,10 @@ function Get-DacFxVersion {
     cd "C:\Program Files\Microsoft SQL Server\150\DAC\bin\"
     $dacfxversion = (./sqlpackage.exe /version)
     return "DacFx $dacfxversion"
-}
\ No newline at end of file
+}
+
+function Get-SwigVersion {
+    ($(swig -version) | Out-String) -match "version (?<version>\d+\.\d+\.\d+)" | Out-Null
```

Comment:
nitpick: this will also work
`(swig -version | Out-String) -match "version (?<version>\d+\.\d+\.\d+)"`

Reply:
the writing of the version you proposed does not carry any changes

Reply:
it simplifies the code, that's why it's just a nitpick